pid_controller/src/pid_controller.cpp

```diff
@@ -474,7 +474,7 @@ controller_interface::return_type PidController::update_and_write_commands(
   const rclcpp::Time & time, const rclcpp::Duration & period)
 {
   // check for any parameter updates
-  update_parameters();
+  param_listener_->try_get_params(params_);
```

Comment:
We can remove update_parameters() method and use just `get_params()` in the `on_configure()` method (not in the RT loop).

Reply:
```suggestion
  param_listener_->try_update_params(params_);
```

Am i missing something or it should be try_get_params?